src/goto-programs/interpreter_evaluate.cpp

```diff
@@ -102,7 +102,9 @@ void interpretert::clear_input_flags()
   }
 }
 
-/// \return Number of leaf primitive types; returns true on error
+/// \param ty: a type
```

Comment:
I'd appreciate a summary on this comment as I don't know what `count_type_leaves` means...